src/plugins/intel_gpu/src/kernel_selector/kernels/ctc_greedy_decoder/ctc_greedy_decoder_kernel_base.cpp

```diff
@@ -19,7 +19,7 @@ JitConstants CTCGreedyDecoderKernelBase::GetJitConstants(const ctc_greedy_decode
 
     if (params.outputs_num == 2) {
         jit.AddConstants({
-            MakeJitConstant("SECOND_OUTPUT_EXIST", 1),
+            MakeJitConstant("LEGACY_MULTIPLE_OUTPUTS", 1),
```

Comment:
I think
LEGACY  : outputs_num > 1 + allow_new_shape_infer  = false case (you cannot check allow_new_shape infer in kernel, but you can check it in combination with number of inputs )
NEW_MULTIPLE_OUTPUT : outputs_num > 1 + allow_new_shape_infer  = true case 
isn't it? 
Always setting outputs_num > 1 as LEGACY_MUL_OUTPUT seems incorrect.

Reply:
Fixed code to use allow_new_shape_infer and number of inputs in the kernel. Thanks for your comments.